src/private/CBCRuntime.h

```diff
@@ -17,24 +17,32 @@
 #import <Foundation/Foundation.h>
 #import <UIKit/UIKit.h>
 
-#pragma mark Breadcrumb retrieval
-
-/** Invokes +catalogBreadcrumbs on the class and returns the corresponding array of strings. */
-FOUNDATION_EXTERN NSArray<NSString *> *CBCCatalogBreadcrumbsFromClass(Class aClass);
-
-/** Invokes +catalogIsPrimaryDemo on the class and returns the BOOL value. */
-FOUNDATION_EXTERN BOOL CBCCatalogIsPrimaryDemoFromClass(Class aClass);
-
-/** Invokes +catalogIsPresentable on the class and returns the BOOL value. */
-FOUNDATION_EXTERN BOOL CBCCatalogIsPresentableFromClass(Class aClass);
-
-/** Invokes +catalogIsDebug on the class and returns the BOOL value. */
-FOUNDATION_EXTERN BOOL CBCCatalogIsDebugLeaf(Class aClass);
+#pragma mark Metadata keys
+
+/** This key represents a boolean value if to present the example in the Catalog app or not */
+FOUNDATION_EXTERN NSAttributedStringKey const CBCIsPresentable;
```

Comment:
Sorry, I didn't mean to copy this type also: please replace NSAttributedStringKey with NSString* throughout.

Also do quick search-and-replace for "breadcrumbs" and replace with `CBCBreadcrumbs`.